index.html

```diff
@@ -1,156 +1,123 @@
-<!DOCTYPE html>
```

Comment:
Hi Alex, 

Can you either remove index.html from your commit or do another git pull for newer version of index.html file? The file you committed is older version so if I approve this pull request, we are going to lose all Khoi's and my work on this file.

Reply:
> Hi Cythia I’ve removed the index.html file :)
> […](#)

Sorry, Alex. I meant to say to update your index.html file.  It seems you remove the html file from the file system. Can you please do git pull from the master and try again?